package.json

```diff
@@ -69,10 +73,25 @@
   "bugs": {
     "url": "https://github.com/paldepind/snabbdom/issues"
   },
+  "remarkConfig": {
+    "plugins": [
+      [
+        "toc",
+        {
+          "tight": true
+        }
+      ]
+    ],
+    "settings": {
+      "listItemIndent": "1",
+      "bullet": "*",
+      "paddedTable": false
+    }
+  },
   "homepage": "https://github.com/paldepind/snabbdom#readme",
   "husky": {
     "hooks": {
-      "pre-commit": "npm test"
+      "pre-commit": "run-s docs check-clean test"
```

Comment:
If a contributor edits the readme (or any markdown file) and forgets to run `npm run docs` and attempts to commit, this will run `npm run docs` and then if that resulted in changes, the commit will fail with just enough information in the error to help understand why:

```text
➜  snabbdom git:(remark) ✗ git commit -m'docs: make some edits'
husky > pre-commit (node v12.10.0)

> snabbdom@0.7.4 docs /home/shahar/src/snabbdom
> remark . --output

README.md: written
examples/carousel-svg/README.md: written

> snabbdom@0.7.4 check-clean /home/shahar/src/snabbdom
> git diff --exit-code

diff --git a/README.md b/README.md
index acce45a..c7d4d5b 100644
--- a/README.md
+++ b/README.md
@@ -57,7 +57,6 @@ performance, small size and all the features listed below.
 
 ## Example
 
-
 ```javascript
 var snabbdom = require('snabbdom');
 var patch = snabbdom.init([ // Init patch function with chosen modules
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! snabbdom@0.7.4 check-clean: `git diff --exit-code`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the snabbdom@0.7.4 check-clean script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/shahar/.npm/_logs/2019-11-26T20_51_24_896Z-debug.log
ERROR: "check-clean" exited with 1.
husky > pre-commit hook failed (add --no-verify to bypass)
➜  snabbdom git:(remark) ✗ 
```